test_package/CMakeLists.txt

```diff
@@ -14,9 +14,7 @@ if (UNIX)
     endif ()
 endif ()
 
-find_package(EXPAT REQUIRED)
+#find_package(EXPAT REQUIRED)
 
 add_executable(example example.c)
-target_link_libraries(example PRIVATE ${EXPAT_LIBRARY})
-target_include_directories(example PRIVATE ${EXPAT_INCLUDE_DIRS})
-target_compile_definitions(example PRIVATE "-DXML_STATIC")
+target_link_libraries(example PRIVATE CONAN_PKG::Expat)
```

Comment:
IMHO, if we do not use the `find_package()` we stop testing one important part of the expat library (the way they export their configuration). In fact, in some other projects I started to use the `cmake_paths` generator instead of the `cmake` one, in order to have a purest CMake code:

https://docs.conan.io/en/latest/integrations/build_system/cmake/cmake_paths_generator.html

Would you mind to try that out?